content/docs/instrumenting/exporters.md

```diff
@@ -178,7 +178,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [JMX exporter](https://github.com/prometheus/jmx_exporter) (**official**)
    * [Munin exporter](https://github.com/pvdh/munin_exporter)
    * [Nagios / Naemon exporter](https://github.com/Griesbacher/Iapetos)
-   * [New Relic exporter](https://github.com/mrf/newrelic_exporter)
+   * [New Relic exporter](https://github.com/corabank/newrelic_exporter.git)
```

Comment:
This should be without the `.git` suffix.

Reply:
Of course, in your time, in the meantime, we will be making other improvements to the chart and improving the documentation.

Reply:
Hum Ok I will remove the .git and update the PR